src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/toolbar.scss

```diff
@@ -2,10 +2,60 @@
 
 $toolbarBackgroundColor: $white;
 $shadowColor: $mischka;
+$itemsBackgroundColor: $alabaster;
+
+$itemsContainerBorder: 1px $gallery solid;
```

Comment:
Please create a separate variable for the usage of `$gallery` like `$itemsBorderColor`, then it will be easier to override in the future.